Realm.podspec

```diff
@@ -30,7 +30,8 @@ Pod::Spec.new do |s|
                               'include/Realm/RLMRealm.h',
                               'include/Realm/RLMResults.h',
                               'include/Realm/RLMSchema.h',
-                              'include/Realm/Realm.h'
+                              'include/Realm/Realm.h',
+			      'include/Realm/RLMRealm_Dynamic.h'
```

Comment:
@segiddins I think the alignment is just github web view... whole file doesn't show that